### PR TITLE
Add argument parsing for Linux module wheels scripts

### PR DESCRIPTION
**Pull request by LucasGandel**
Add argument parsing allowing users to:
- Forward CMake options to the module
- Add option to exclude libs from repaired wheel on Linux:
  Some shared libraries like the CUDA driver library must be provided at
  runtime. Hence, it must be excluded from the repaired wheel (while it
  must be provide in LD_LIBRARY_PATH for the project to compile).

The optional python version is still sent as positional argument for
backward compatibility.

### CODE REVIEW

**Comment by LucasGandel** [comment]
@thewtex @tbirdso Can you please have a quick look when you have time?

This adds argument parsing on Linux to:
- forward CMake options to modules
- exclude shared libraries from the repaired wheel.

**Comment by LucasGandel** [comment]
> LGTM, but would be good for @thewtex to have eyes on these changes as well before merge.
Thanks a lot @tbirdso! Of course I'll wait for Matt's approval before merging. I am also very open to changing the approach I chose for argument parsing. I don't have much experience in shell scripting.

**Review by thewtex — APPROVED** [review]
@LucasGandel very awesome!!

Looking forward to the CUDA wheels!

**Pull request merged**
